pip/wheel.py

```diff
@@ -668,19 +668,24 @@ def _build_one(self, req, output_dir):
                     logger.info('Stored in directory: %s', output_dir)
                     return wheel_path
                 except:
-                    return None
+                    pass
```

Comment:
You may be right - presumably even if the package is installed, the user wants to know that the wheel build failed as the implication is that the expected improved performance on repeated installs will be lost. Maybe there should even be a warning "wheel build failed, falling back to source install" as well as the debug info providing the full exception details.